shared/menubar/index.desktop.js

```diff
@@ -46,22 +47,23 @@ class MenubarRender extends React.Component<Props, State> {
   state: State = {showingMenu: false}
   attachmentRef = React.createRef<Kb.Icon>()
 
-  _refreshUserFileEditsIfPossible = () =>
+  _refreshUserFileEditsOrWaitForKbfsDaemon = () =>
     this.props.loggedIn &&
-    this.props.kbfsDaemonStatus.rpcStatus === 'connected' &&
-    this.props.refreshUserFileEdits()
+    (this.props.kbfsDaemonStatus.rpcStatus === 'connected'
+      ? this.props.refreshUserFileEdits()
+      : this.props.waitForKbfsDaemon())
```

Comment:
Note that this alone won't trigger the actual load. I think you also want `kbfsDaemonRpcStatusChanged` [here](https://github.com/keybase/client/blob/f3cca64128fe659bcdafa144720488b60a04de8e/shared/actions/fs/platform-specific.desktop.js#L394) so that once we're connected, we'd load it. Similar to [this](https://github.com/keybase/client/blob/f3cca64128fe659bcdafa144720488b60a04de8e/shared/actions/fs/platform-specific.desktop.js#L388-L390).